lib/generators/module/templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Vsp
```

Comment:
Ours is `README.md`.  All the other existing Engines have a `README.rdoc`.  Do we want to standardize on one?